Bottom: 30c50efcbff260548fc311d81a94810ad9b678ad
Top:    89da7110d11eacd6bc5744af902e2e24e8a49539
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 10:41:37 +0800

Refresh of debug.patch

---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index aa7ae07..adf302a 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -1295,6 +1295,16 @@ int parse_mount(struct autofs_point *ap, const char *name,
 
 	p = skipspace(pmapent);
 
+	/*
+	 * Change special map -hosts into <type>: format so it
+	 * doesn't confuse the options parse.
+	 */
+	if (strstr(p, "-hosts")) {
+		char *pos = p;
+		*pos++ = 'h'; *pos++ = 'o'; *pos++ = 's'; *pos++ = 't';
+		*pos++ = ':';
+	}
+
 	/* Deal with 0 or more options */
 	if (*p == '-') {
 		char *tmp, *mnt_options = NULL;
